views/forms/ControlledSelect/index.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import { TextField } from '@mui/material'
 import { useController } from 'react-hook-form'
 
-const ControlledSelectInput = ({ name, control, value, sx, ...rest }) => {
-  const { field } = useController({ name, control })
+const ControlledSelectInput = ({ control, value, sx, ...rest }) => {
+  const { field } = useController({ name: rest.name, control })
```

Comment:
🎱 if we need `name`, should we destructure it out like the other props?

_we'd need to add `name={name}` on the `TextField` below_
```suggestion
const ControlledSelectInput = ({ control, name, value, sx, ...rest }) => {
  const { field } = useController({ control, name })
```

Reply:
[de58fd1](https://github.com/AMP-SCZ/dpdash/pull/561/commits/de58fd1cf84eacd817827865597d1097ede05d91)